extensions/adobe/experience/analytics/context-data.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/analytics/context-data",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "context-data",
+  "type": "object",
+  "description": "A free form field where Analytics context data will be captured.",
+  "definitions": {
+    "context-data-datatype": {
+      "properties": {
+        "xdm:context-data-datatype": {
```

Comment:
We do not support "-" in the property names. Also, it seems like you are creating a map object to send context data so having a suffix of "datatype" in the object name would not make sense. I would suggest to change the property name to "xdm:contextData"

Reply:
thank you, I'm adding that